collect_app/src/main/java/org/odk/collect/android/instancemanagement/send/ReadyToSendBanner.kt

```diff
@@ -0,0 +1,45 @@
+package org.odk.collect.android.instancemanagement.send
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.lifecycle.LifecycleOwner
+import org.odk.collect.android.databinding.ReadyToSendBannerBinding
+import org.odk.collect.strings.R
+
+const val ONE_SECOND = 1000L
+const val ONE_MINUTE = 60000L
+const val ONE_HOUR = 3600000L
+const val ONE_DAY = 86400000L
+
+class ReadyToSendBanner(context: Context, attrs: AttributeSet?) : ConstraintLayout(context, attrs) {
+    constructor(context: Context) : this(context, null)
+
+    private val binding = ReadyToSendBannerBinding.inflate(LayoutInflater.from(context), this, true)
+
+    fun init(viewModel: ReadyToSendViewModel, owner: LifecycleOwner) {
```

Comment:
Bit nit-picky here, but I think it would slightly cleaner to have a method like `setData(numberOfInstancesReadyToSend: Int, numberOfSentInstances: Int, lastInstanceSentTimeMillis: Long)` here and then handle the observation in the Activity. That means the `View` doesn't need to know anything about the ViewModel (or `LiveData`) which makes it simpler, more reusable and (because of that) easier to test.

Reply:
Fixed.